tools/cli/workflowCommands.go

```diff
@@ -203,6 +203,15 @@ func startWorkflowHelper(c *cli.Context, shouldPrintProgress bool) {
 	if c.IsSet(FlagCronSchedule) {
 		startRequest.CronSchedule = common.StringPtr(c.String(FlagCronSchedule))
 	}
+	if c.IsSet(FlagRetryAttempts) {
+		interval := int32(c.Int(FlagRetryInterval))
+		attempts := int32(c.Int(FlagRetryAttempts))
+		startRequest.RetryPolicy = &s.RetryPolicy{
+			MaximumAttempts:          common.Int32Ptr(attempts),
```

Comment:
I think in some use cases user may want to retry a workflow for certain period of time and don't care much about the exact number of attempts. In that case, `ExpirationInterval` might be more useful.

Reply:
Agreed, this could be useful. Updated code.